include/sys/proc.h

```diff
@@ -58,6 +59,7 @@ struct proc {
   sigaction_t p_sigactions[NSIG]; /* (@) description of signal actions */
   condvar_t p_waitcv;             /* (a) processes waiting for this one */
   int p_exitstatus;               /* (@) exit code to be returned to parent */
+  vnode_t *cwd;                   /* Current working directory */
   /* program segments */
   vm_segment_t *p_sbrk; /* ($) The entry where brk segment resides in. */
```

Comment:
* You should add information about `cwd`'s locking strategy (e.g. `(@)`, `(a)`, `($)`).
* You should prefix fields inside `struct proc` with `p_`.